endor/src/error/InvalidCredentialsException.js

```diff
@@ -0,0 +1,16 @@
+import { getActiveLogger } from '../utils/winston';
+
+const log = getActiveLogger();
+
+export default class InvalidCredentialsException {
+  constructor(message) {
+    this.message = message;
+    this.type = 'Invalid Credentials';
+    this.status = 400; // TODO I don't know about this
```

Comment:
401 would probably be a good error code here. 

https://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses